gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@
 # see https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note
 [versions]
 checker = "3.24.0"
-ideProbe = "0.41.0+28-68889008-SNAPSHOT"
+ideProbe = "0.0.0+530-68889008-SNAPSHOT"
```

Comment:
Huh weird... looks that ide-probe is cloned without git tags, and/or your fork doesn't include the tags from the original VirtusLab/ide-probe repo?

Reply:
Some combination of `git fetch --tags` and/or `git push --tags` should help

Reply:
(albeit this is of minor importance anyway)

Reply:
Ok, my fork has tags included - at leas it works for me locally. Won't fix it if no need for now

Reply:
To be updated to 0.43.0 once released

Reply:
If `0.43.0` will have https://github.com/VirtusLab/ide-probe/pull/257 included